presto-benchmark/src/main/java/com/facebook/presto/benchmark/HashBuildAndJoinBenchmark.java

```diff
@@ -81,7 +81,7 @@ from lineitem join orders using (orderkey)
         driversBuilder.add(hashBuilder);
         DriverFactory hashBuildDriverFactory = new DriverFactory(0, true, false, driversBuilder.build(), OptionalInt.empty());
         Driver hashBuildDriver = hashBuildDriverFactory.createDriver(taskContext.addPipelineContext(0, true, false).addDriverContext());
-        hashBuildDriverFactory.close();
+        hashBuildDriverFactory.noMoreDriver();
```

Comment:
`noMoreDrivers`?

Also, commit message incorrectly refers to `noMoreOperators`